Automattic/wp-calypso
# Unify loading screens in all onboarding flows

#### Proposed Changes

- [Persist](https://github.com/Automattic/wp-calypso/blob/c04d343e7c51ec3bc0f180892ab388c554601996/client/landing/stepper/declarative-flow/internals/index.tsx#L43) the `intent` using STEPPER_INTERNAL_STORE since the 'goals' and 'intent' value will be cleared when exitFlow, make they not available on processing-step
- Update [loading](https://github.com/Automattic/wp-calypso/blob/893a32d4c2bc5749f276fcc645dca522a7fbd3e9/client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx#L51) strings for difm flow
- Update [loading](https://github.com/Automattic/wp-calypso/blob/893a32d4c2bc5749f276fcc645dca522a7fbd3e9/client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx#L59) strings for sell flow
- Update [position](https://github.com/Automattic/wp-calypso/blob/893a32d4c2bc5749f276fcc645dca522a7fbd3e9/client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/style.scss#L120) of the loading indicator for the /woo-confirm step
- Fix [progress bar overlap](https://github.com/Automattic/wp-calypso/blob/893a32d4c2bc5749f276fcc645dca522a7fbd3e9/client/landing/stepper/declarative-flow/internals/global.scss#L74) on import step
#### Testing Instructions
| Flow  | Record | Loading messages |
| :- | -----:| -----:|
| Goals -> Hire a pro...  | <details> <video src="https://user-images.githubusercontent.com/10071857/181130384-ea45b3d9-a91a-4491-87d7-74ba637c6414.mp4"> </details> | Securing your data <br> Enabling encryption <br> Applying a shiny top coat <br> Closing the loop |
| Goals -> Sell flow  | <details> <video src="https://user-images.githubusercontent.com/10071857/181130662-8a9d7554-c9b2-4d80-9f54-9fbf58aa6c3d.mp4"> </details> | Sprinkling some magic <br> Securing your data <br> Enabling encryption <br> Applying a shiny top coat <br> Closing the loop |
| Goals -> Sell flow -> Pro -> woo-confirm  | <details> <video src="https://user-images.githubusercontent.com/10071857/181130168-e888edaa-14d7-4a06-96dd-8563f045b8db.mp4"> </details> | NA |
| Goals -> Import flow  | <details> <video src="https://user-images.githubusercontent.com/10071857/181129829-5af40a06-acbb-41ba-be42-3072fee936c2.mp4"> </details> | N/A |

cc: @saygunnyc 
Related to #64960
